apps/fluent-tester/src/RNTester/TestComponents/Text/CustomizeUsage.tsx

```diff
@@ -23,7 +22,7 @@ export const CustomizeUsage: React.FunctionComponent<{}> = () => {
   const Wingdings = Text.customize({ tokens: { variant: 'captionStandard', fontFamily: 'Wingdings' } });
 
   return (
-    <View style={styles.root}>
```

Comment:
Any particular reason we're getting rid of the style?

Reply:
The root from common styles defines the flexDirection as "row" when the previous flow was "column". Removing the styles returns the layout to "column".

Reply:
Do the margin values not make a visual difference?

Reply:
It does a bit but it doesn't effect the example too much. I can add it back in if you want. I just made this change to make the Text examples more consistent with the other examples which use the styles from common/styles.

Reply:
oh no it's fine, consistency is great.